packages/components/src/item-group/styles.ts

```diff
@@ -17,6 +17,11 @@ export const unstyledButton = css`
 
 	&:hover {
 		color: ${ COLORS.ui.theme };
+
+		svg,
+		path {
+			fill: ${ COLORS.ui.theme };
+		}
```

Comment:
Perhaps this could be moved to outside of the `:hover` state, so that the `fill` applies on any state when the `color` is changed? e.g. these new CSS lines would be on the same level as `&:hover` and `&:focus`.

Reply:
If we move these new lines outside the `:hover` state, I believe this PR will also cover https://github.com/WordPress/gutenberg/issues/50757.

Reply:
I would rather keep the states separate (:hover, :active, :focus and no state at all). But the issue you mentioned I can take a look at it 😄

Reply:
I'm not sure there's likely to be a case where we don't want the icons to inherit the `color` from the button, regardless of the button state. Keeping the states separate sounds like it may introduce some unnecessary duplication here.

We could also use `currentColor` rather than `COLORS.ui.theme`, which would ensure the same active color is used for all states ([here's an example](https://osvaldas.info/keeping-css-short-with-currentcolor/#svg)).

Reply:
I think you just sold this idea to me 😁
I am away couple days (living life outside of coding). Most likely Sunday I have some to implement this